contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,146 @@
+
+```
+identify(email)                                   # in the frontend
+alias(backend_unique_id, email)      # in the backend - THIS WILL NOT WORK
+alias(email, backend_unique_id2)    # in the backend - the right order
+```
+After updating to 1.41.0, check [ingestion warnings](#new-ingestion-warnings) and run async migrations 0005, 0006 and 0007.
+
+## PostHog 1.41.0 release notes
+
+**Release highlights:**
+
+- New: Persons on events on by default
+- New: Count of events per user
+- New: Text cards on dashboards
+- New: App metrics
+- New: Ingestion warnings
+- New: Change your email yourself
+- New: Hedgehog mode
+- Improved: Recordings interface
+- One more thing: Site apps 
+
+### New: Persons on events on by default
+
+We used to store events in one table and persons in another table. That meant that, once you reached billion event scale, any query which touched person properties would time out. But, no longer! After running [an extensive beta since update 1.39.0](/blog/the-posthog-array-1-39-0#beta-improving-query-performance-by-combining-persons-and-events), we've now added person data onto the events themselves. This change also applies to groups, adding group properties and aggregation to the same table. 
+
+You won’t see any UI changes as a result of this change — persons will still have their own Persons & Groups section on the sidebar, for example — but you will notice results are a lot (up to 400%!) faster for any queries involving persons, groups or events. This is a massive change, so be sure to [read the full announcement for more info](/blog/persons-on-events).
+
+> **Note:** As a result of putting persons on events we now have to be strict about identified and unidentified users. Now, only anonymous users can be aliased into other users - meaning that once you call `identify` on a user, you can no longer alias the user into another identified user. Because of that, we recommend double checking your `identify` and `alias` calls, and following advice above [how to upgrade a self-hosted instance to 1.41.0](/blog/the-posthog-array-1-41-0). 
+
+### New: Count of events per user
+
+![count events per user](../images/blog/array/1-41-0-count-events-per-user.png)
+
+Have you ever asked yourself '*What's the average average number of purchases per user?*' or '*What's the maximum number of forms submitted per user?'* 
+
+Questions like these used to be hard to answer with PostHog, but no more! We've added a new series option for insights, enabling you to see the count of events per user - including sub-options for seeing the averages, minimum and maximum count of events per user. 
+
+**Bonus update:** While working on this feature, we also took the opportunity to improve our math selector UI.
+
+### New: Text cards on dashboards
+
+![text cards on dashboards](../images/blog/array/1-41-0-text-card-on-dashboard.png)
```

Comment:
I'd use a pie chart there instead of trend as we can't tell who's a yes and who's a no on it & maybe mention the source is our users i.e. https://posthog.com/404